autofix-go/replace-to-replaceAll.batch.yml

```diff
@@ -0,0 +1,18 @@
+name: replace-to-replaceAll
+description: |
+  This batch change uses [Comby](https://comby.dev) to replace the deprecated String.replaceAll(a,b,c,-1) by String.Replace(a,b,c)
+
+on:
+  - repositoriesMatchingQuery: .Replace(..., ..., -1) lang:go patternType:structural
+
+steps:
+  - run: comby -in-place '.Replace(:[a],:[b],:[c], -1)' '.ReplaceAll(:[a],:[b],:[c])' .go -matcher .go -exclude-dir .,vendor
```

Comment:
I would probably add `strings.Replace` to the actual pattern, otherwise you run the (maybe unlikely) danger of changing code for a `Replace` function that someone defined themselves, like `my.Replace`, which would be a false positive.